mosaic-runtime/src/commonMain/kotlin/com/jakewharton/mosaic/ui/TextStyle.kt

```diff
@@ -1,35 +1,80 @@
+@file:Suppress("NOTHING_TO_INLINE")
+
 package com.jakewharton.mosaic.ui
 
 import androidx.compose.runtime.Immutable
 import androidx.compose.runtime.Stable
+import kotlin.jvm.JvmInline
 
 @Immutable
-public class TextStyle private constructor(
-	private val bits: Int,
+@JvmInline
+public value class TextStyle internal constructor(
+	@PublishedApi
+	internal val bits: Int,
 ) {
 	public operator fun plus(other: TextStyle): TextStyle = TextStyle(bits or other.bits)
-	public operator fun contains(style: TextStyle): Boolean = (style.bits and bits) != 0
+	public operator fun contains(other: TextStyle): Boolean = (bits and other.bits) == other.bits
 
 	public companion object {
 		@Stable
-		public val None: TextStyle = TextStyle(0)
+		public val Null: TextStyle = TextStyle(NullTextStyle)
+
+		@Stable
+		public val None: TextStyle = TextStyle(NoneTextStyle)
```

Comment:
What are the semantics here? Null resets to no style and none is a no-op? Or the opposite maybe? I think we need better names here, and definitely some docs.